actions/mouths-of-watersheds/mouths-of-watersheds.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import csv
+from netCDF4 import Dataset
+from shapely.geometry import shape
+import geojson
+from mow.helpers import *
+from mow.geo_data_grid_2d.vic import VicDataGrid
+
+parser = argparse.ArgumentParser('Output a CSV file with the lon lat of the mouth of each watershed')
+parser.add_argument('nc', help='an RVIC-netCDF file')
```

Comment:
We should name things after what they are, not what their type is. `nc` just means NetCDF -- it could contain anything. A name like `flow_direction_file` is more descriptive.